gef.py

```diff
@@ -4231,9 +4231,12 @@ def get_ctypes_value(self, struct, item, value):
         default = ""
         for name, values in values_list:
             if name != item: continue
-            for val, desc in values:
-                if value == val: return desc
-                if val is None: default = desc
+            if type(values) == list:
```

Comment:
You're testing that it's a list, but don't test that it's a list of 2-tuples. I think that gives extra reason to instead just test that it's callable and catch if it later on doesn't fit `[(a,b),...]`